### PR TITLE
LRQA-53956 | Master

### PR DESCRIPTION
**Pull request by nkkwon**
https://issues.liferay.com/browse/LRQA-53956
AppBuilder#SearchForDeployedItem

### CODE REVIEW

**Comment by nkkwon** [comment]
ci:test:sf

**Comment by nkkwon** [comment]
ci:test:relevant

File: modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase [hunk 1]
```diff
@@ -1446,6 +1446,104 @@ definition {
 		AppBuilderAdmin.validateItemNameOnTable(itemName = "Content of form deployed as widget.");
 	}
 
+	@description = "Search for a deployment in the deployment lists"
+	@priority = "5"
+	test SearchForDeployedItem {
+		property portal.acceptance = "false";
```
Comment (alexcandidos) [inline modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase]: Hey @nkkwon , could you please just remove this line?

**Comment by alexcandidos** [comment]
ci:test:app-builder

**Comment by alexcandidos** [comment]
Pull request forwarded to https://github.com/brianchandotcom/liferay-portal/pull/84083. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by alexcandidos** [comment]
Thanks @nkkwon !

**Pull request closed without merging**